core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -34,10 +34,19 @@ oppia.directive('learnerDashboardIcons', [
         'LearnerDashboardActivityIdsObjectFactory',
         'LearnerPlaylistService',
         function(
-            $scope, LearnerDashboardIdsBackendApiService,
-            LearnerDashboardActivityIdsObjectFactory,
-            LearnerPlaylistService) {
+          $scope, LearnerDashboardIdsBackendApiService,
```

Comment:
No, keep the indentation as it was. It was done like that to differentiate it from line 40 where the new inner scope actually starts.

Reply:
Them the file is unable to push because it fails the eslint.

Reply:
Ah, really? OK, I retract my comment, then. We'll have to look into the eslint thing at some point, but let's not that hold up your PR.